### DISCUSSION TITLE
Welcome to atala-prism-schema Discussions!

### DISCUSSION THREAD

**Discussion by zakaio** [zakaio/atala-prism-schema#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by rodolfomiranda** [comment]
Rodolfo Miranda here from [RootsID](www.rootsid.com). We develop open source building blocks and tools for the SSI Cardano community and in general. Great initiative! 
For reference to other here is the link to the repo Verifiable [Credential Schema Catalog](https://github.com/roots-id/vc-schema-catalog) where we consolidated well known schemas from several industries.
